src/execute.rs

```diff
@@ -0,0 +1,35 @@
+use volta_core::error::report_error;
+use volta_core::log::{LogContext, LogVerbosity, Logger};
+use volta_core::run::execute_tool;
+use volta_core::session::{ActivityKind, Session};
+use volta_core::signal::setup_signal_handler;
+use volta_fail::ExitCode;
+
+#[inline]
+pub(super) fn run_shim() {
+    Logger::init(LogContext::Shim, LogVerbosity::Default)
+        .expect("Only a single Logger should be initialized");
```

Comment:
Is there any reason besides extra loggers being set up why this would fail? My concern is that dying with `"Only a single Logger should be initialized"` might be misleading if there *are* other reasons it would fail.

Reply:
(I note that this is just copied over to support the change – so no worries about making any change here *for this PR* even if we do think we should.)

Reply:
Per the [docs for 'log'](https://docs.rs/log/0.4.8/log/fn.set_boxed_logger.html), that is the only failure path for the log initialization, yes.

Reply:
Awesome! Thank you!